src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -227,7 +227,21 @@ public WindowTransparencyLevel TransparencyLevel
 
         private WindowTransparencyLevel EnableBlur(WindowTransparencyLevel transparencyLevel)
         {
-            if(DwmIsCompositionEnabled(out var compositionEnabled) != 0 || !compositionEnabled)
+            var version = RtlGetVersion();
```

Comment:
I wonder if we shouldn't expose such information on our platform level somehow. Since we might want to feature flag more features based on the OS/Composition/Remote Desktop.

Reply:
Does not have to be done as a part of this PR but it might be useful to have it.

Reply:
cached and made static property on win32platform.